packages/features/users/components/UserTable/BulkActions/MassAssignAttributes.tsx

```diff
@@ -138,9 +158,75 @@ function SelectedAttributeToAssign() {
   );
 }
 
-export function MassAssignAttributesBulkAction({ table, filters }: Props) {
-  const { selectedAttribute, setSelectedAttribute, foundAttributeInCache } = useSelectedAttributes();
-  const [selectedAttributeOptions, setSelectedAttributeOptions] = useSelectedAttributeOption();
+function Content({ showMultiSelectWarning }: { showMultiSelectWarning: boolean }) {
```

Comment:
I've extracted `Content` out of the main component. The implementation is almost the same.